fs/span-manager/span_manager.go

```diff
@@ -163,6 +170,28 @@ func (m *SpanManager) buildAllSpans() {
 	}
 }
 
+// fetchWithRetries fetches the requested data and verifies that the span digest matches the one in the ztoc m.retries times.
+// It does not retry when there is an error fetching the data, because retries already happen lower in the stack in httpFetcher.
+// If there is an error fetching data from remote, it is not an transient error.
+func (m *SpanManager) fetchWithRetries(spanID compression.SpanID, buf []byte, offset compression.Offset) error {
+	var (
+		err error
+		n   int
+	)
+	for i := 0; i < m.retries; i++ {
```

Comment:
If ``m.retries`` is set to 0, which is no retry, we do not fetch at all? That doesn't sound right.

Reply:
My comment relates to a comment from @sparr above about when user actually specifically wants to set retry to 0 and yet we convert 0 to the default value.

Reply:
Fair enough. I've renamed it to maxSpanVerificationTries, which now defaults to 1 + (number of retries requested). It will only default to `defaultRetries` if it's still less than or equal to 0.